cosmrs/src/crypto/public_key.rs

```diff
@@ -1,6 +1,7 @@
 //! Public keys
 
 use crate::{prost_ext::MessageExt, proto, AccountId, Error, ErrorReport, Result};
+use cosmos_sdk_proto::cosmos;
```

Comment:
`cosmos_sdk_proto` is imported as `proto` above, so you don't need an import here.

Instead you can use `proto::cosmos`

Reply:
Done